genesyscloud/architect_schedulegroups/resource_genesyscloud_architect_schedulegroups.go

```diff
@@ -3,26 +3,46 @@ package architect_schedulegroups
 import (
 	"context"
 	"fmt"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-	"github.com/mypurecloud/platform-client-sdk-go/v121/platformclientv2"
 	"log"
 	"strings"
 	"terraform-provider-genesyscloud/genesyscloud/consistency_checker"
 	"terraform-provider-genesyscloud/genesyscloud/provider"
 	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
 	"terraform-provider-genesyscloud/genesyscloud/util"
 	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/retry"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v121/platformclientv2"
 )
 
 /*
 The resource_genesyscloud_architect_schedulegroups.go contains all of the methods that perform the core logic for a resource.
 */
 
-// getAllAuthArchitectSchedulegroups retrieves all of the architect schedulegroups via Terraform in the Genesys Cloud and is used for the exporter
-func getAllAuthArchitectSchedulegroupss(ctx context.Context, clientConfig *platformclientv2.Configuration) (resourceExporter.ResourceIDMetaMap, diag.Diagnostics) {
```

Comment:
@dginty4 Lets talk tomorrow during stand up.  This was kind of an ugly miss as both Hemanth and reviewed this code and I reviewed this and missed this.

We probably want a ticket out there to move the logic here over to the proxy.  I also want to chat with Monisha about getting a test case around this.  

We should also talk about how quickly we can get a point release out there to fix this because this breaks the export entirely for Architect schedule groups